### DISCUSSION TITLE
First-time contributor workflow approval buttons disappeared

### DISCUSSION THREAD

**Discussion by cjmayo** [community/community#30694] [Pull Requests]
On a couple of PRs from first-time contributors:
https://github.com/linkchecker/linkchecker/pull/628
https://github.com/linkchecker/linkchecker/pull/629

There is no button to approve the checks workflow.

The checks tab has "Workflow runs completed with no jobs".

Looks like maybe where the checks weren't approved for a period of time an empty workflow ran and now the checks are in limbo "Some checks haven’t completed yet".

**Comment by Pr0methean** [comment]
This also happened to https://github.com/zip-rs/zip2/pull/352, although pushing an empty commit to the destination branch and then clicking "Update branch" seems to have fixed it.
